test/e2e/azure/account.go

```diff
@@ -0,0 +1,124 @@
+package azure
+
+import (
+	"encoding/json"
+	"log"
+	"os/exec"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
+// Account holds the values needed to talk to the Azure API
+type Account struct {
```

Comment:
Right now its about porting what is currently being done in the bash scripts to something a bit more manageable.